doc/nrf/protocols/esb/index.rst

```diff
@@ -323,6 +323,8 @@ For the PTX node, this switching occurs after transmitting a packet and before w
 For the PRX node, this switching occurs after receiving a packet and before transmitting an acknowledgment (RX -> TX).
 Enabling this feature can improve the responsiveness and efficiency of the radio communication system by reducing latency.
 
+Enabling :kconfig:option:`CONFIG_ESB_FAST_CHANNEL_SWITCHING` for the nRF54H20 SoC allows radio channel switching in RX radio mode without transitioning to the radio's DISABLE state.
```

Comment:
```suggestion
Enabling the :kconfig:option:`CONFIG_ESB_FAST_CHANNEL_SWITCHING` Kconfig for the nRF54H20 SoC allows radio channel switching in RX radio mode without transitioning to the ``DISABLE`` mode.
```
Or is the mode DISABLED?

Reply:
Good point, should be DISABLED. 
I used `state` instead of `mode` acording to SOC documentation.